README.md

```diff
@@ -7,6 +7,18 @@ A Kubernetes Operator to automate deployment of
 [Cryostat](https://github.com/cryostatio/cryostat) and provide an
 API to manage [JDK Flight Recordings](https://openjdk.java.net/jeps/328).
 
+## SEE ALSO
+
+* [cryostat-core](https://github.com/cryostatio/cryostat-core) for
+the core library providing a convenience wrapper and headless stubs for use of
+JFR using JDK Mission Control internals.
+
+* [cryostat-operator](https://github.com/cryostatio/cryostat-operator)
```

Comment:
Seems a little funny to link the Operator back to itself here, no?

Should there be a link to `cryostat-web` instead?

Links to `jfr-datasource`, `cryostat-grafana-dashboard` here could also make a lot of sense since the operator also deploys those images.